src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -261,6 +262,11 @@ private static IUmbracoBuilder AddHttpClients(this IUmbracoBuilder builder)
                 ServerCertificateCustomValidationCallback =
                     HttpClientHandler.DangerousAcceptAnyServerCertificateValidator,
             });
+        builder.Services.AddHttpClient(Constants.HttpClients.WebhookFiring, (services, client) =>
+        {
+            var productVersion = services.GetRequiredService<IUmbracoVersion>().SemanticVersion.ToSemanticStringWithoutBuild();
+            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("Umbraco", productVersion));
```

Comment:
Final suggestion: we apparently already use `Umbraco-CMS` as User-Agent product name when querying OEmbed endpoints:
https://github.com/umbraco/Umbraco-CMS/blob/56486b1ace877bb0dc11b807dce713c1a1534a84/src/Umbraco.Core/Media/EmbedProviders/OEmbedProviderBase.cs#L58

And technically that is the correct product name as well, right?
```suggestion
            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("Umbraco-CMS", productVersion));
```